internal/cli/atlas/deployments/search/indexes/create.go

```diff
@@ -38,7 +38,7 @@ import (
 const (
 	namePattern        = "^[a-zA-Z0-9][a-zA-Z0-9-]*$"
 	connectWaitSeconds = 10
-	createTemplate     = "Search index created\n"
+	createTemplate     = "Search index created with ID {{.IndexID}}\n"
```

Comment:
Nit `:` or ID at the end

Reply:
will add this to my next PR to avoid delays Thank you!